src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -91,6 +92,12 @@ class EmailServiceTest {
   void setUp() {
   }
 
+  @AfterEach
```

Comment:
These were leaking.

Reply:
Just re-ran and can definitely remove these.  I wonder if there was something else happening that made it look like these weren't being released.